Source/SelfService/Web/microservice/microserviceView/healthStatus/dataTable.tsx

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import React, { useCallback, useState } from 'react';
+
+import { ContainerStatusInfo } from 'Source/SelfService/Web/api/api';
+
+import { Box, Paper, Typography } from '@mui/material';
+import { ExpandMore, ExpandLess } from '@mui/icons-material';
+import { DataGridPro, GridRowId } from '@mui/x-data-grid-pro';
+
+import { PodLogScreen } from '../../podLogScreen';
+import { columns } from './dataTableColumns';
+
+const styles = {
+    podTitle: {
+        fontWeight: 500,
+        lineHeight: '1.5rem',
+        letterSpacing: '0.17px',
+        minHeight: 5.75,
+        p: 1.25,
+        borderBottom: '1px solid rgba(14, 13, 16, 1)'
+    },
+    dataTableWrapper: {
+        'mb': 3,
+        '& .negativeRowSpanHack': {
+            mr: 6.25,
+        },
+        '& .MuiDataGrid-columnHeader[data-field="__detail_panel_toggle__"]': {
+            display: 'none'
+        }
+    },
+    dataTable: {
+        '& .MuiDataGrid-row': {
+            cursor: 'default'
+        }
+    }
+};
+
+const DetailPanelExpandIcon = () => <ExpandMore fontSize='medium' />;
+const DetailPanelCollapseIcon = () => <ExpandLess fontSize='medium' />;
+const CustomToolbar = (rows: Rows[]) =>
+    <Typography variant='body2' sx={styles.podTitle}>{`Pod: ${rows[0]?.podName || 'N/A'}`}</Typography>;
+
+type Rows = {
+    id: string
+    podName: string
+    containerName: string
+    application: string
+    state: string
+    age: string
+    image: string
+    started: string
+    restarts: number
+};
+
+type Row = {
+    row: Rows
+}
```

Comment:
I am a little confused by these `Rows` and `Row` - what are these types? Maybe they can get some different names to explain to me what they are?